### PR TITLE
Fix unit test failures

### PR DESCRIPTION
**Pull request by ChanikaRuchini**
## Purpose
$subject

### CODE REVIEW

File: components/org.wso2.carbon.identity.conditional.auth.functions.test.utils/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/test/utils/sequence/JsSequenceHandlerRunner.java [hunk 1]
```diff
@@ -103,6 +108,9 @@ public void init(URL applicationAuthenticatorConfigFileLocation) throws Invocati
         jsFunctionRegistry = new JsFunctionRegistryImpl();
         FrameworkServiceDataHolder.getInstance().setJsFunctionRegistry(jsFunctionRegistry);
 
+        jsExecutionSupervisor = new JSExecutionSupervisor(THREAD_COUNT,SUPERVISOR_TIMEOUT);
```
Comment (janakamarasena) [inline components/org.wso2.carbon.identity.conditional.auth.functions.test.utils/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/test/utils/sequence/JsSequenceHandlerRunner.java:111]: Formatting
```suggestion
        jsExecutionSupervisor = new JSExecutionSupervisor(THREAD_COUNT, SUPERVISOR_TIMEOUT);
```

File: components/org.wso2.carbon.identity.conditional.auth.functions.user/pom.xml [hunk 2]
```diff
@@ -93,7 +93,10 @@
             <groupId>org.wso2.carbon.identity.framework</groupId>
             <artifactId>org.wso2.carbon.identity.core</artifactId>
         </dependency>
-        
+        <dependency>
```
Comment (janakamarasena) [inline components/org.wso2.carbon.identity.conditional.auth.functions.user/pom.xml:96]: Why is this dependency added?

File: components/org.wso2.carbon.identity.conditional.auth.functions.test.utils/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/test/utils/sequence/JsSequenceHandlerRunner.java (hunk 1 above)
Comment (ChanikaRuchini) [reply components/org.wso2.carbon.identity.conditional.auth.functions.test.utils/src/main/java/org/wso2/carbon/identity/conditional/auth/functions/test/utils/sequence/JsSequenceHandlerRunner.java]: resolved

File: components/org.wso2.carbon.identity.conditional.auth.functions.user/pom.xml (hunk 2 above)
Comment (ChanikaRuchini) [reply components/org.wso2.carbon.identity.conditional.auth.functions.user/pom.xml]: resolved

**Review by janakamarasena — APPROVED** [review]

**Pull request merged**